teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -9,7 +9,8 @@
     "Authors:\n",
```

Comment:
<p><code>open</code>  mode should be <code>wb</code>.  In append mode (<code>a</code>)  you would add data to the file in successive executions, which we don't want! Also, since we are using the <code>with</code>  context manager, you don't need <code>f.close()</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>done</p>